### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

**Comment by Betcheg** [comment]
The man Apollo needed.
@NASA hire this dude

File: BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s [hunk 1]
```diff
@@ -890,7 +890,7 @@ REP40ALM	CAF	V05N09		# (14)
 		TCF	+2		# PROCEED 	CHECK FOR P42
 		TCF	REP40ALM	# V32E		REDISPLAY ALARM
 
-		INDEX	WHICH		# FOR P42, ALLOW CREW TO PRECEED EVEN
+		INDEX	WHICH		# FOR P42, ALLOW CREW TO PRECEDE EVEN
```
Comment (Link2Twenty) [inline BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s]: PROCEED

**Comment by hamidr** [comment]
Are you sure this is not going to crash the system?

**Comment by giraam** [comment]
Conquering the universe one type at a time

**Comment by rafaelcaricio** [comment]
LGTM

File: CONIC_SUBROUTINES.s [hunk 2]
```diff
@@ -106,7 +106,7 @@
 #							RESPECTIVELY, IS THE CENTRAL BODY
 #	TAU		+28			DESIRED TRANSFER TIME IN CENTISECONDS (DP)
 #							MAY BE POS OR NEG AND ABSOLUTE VALUE MAY BE GREATER OR LESS THAN ONE ORBITAL PERIOD.
-#	XKEPNEW		+17 FOR EARTH		DP GUESS OF ROOT X OF KEPLERS EQN IN SQRT(METERS).SIGN SHOULD AGREE WTIH THAT OF TAU.
+#	XKEPNEW		+17 FOR EARTH		DP GUESS OF ROOT X OF KEPLERS EQN IN SQRT(METERS).SIGN SHOULD AGREE WITH THAT OF TAU.
```
Comment (marcosalles) [inline CONIC_SUBROUTINES.s]: `WTIH` again?

File: CONIC_SUBROUTINES.s [hunk 3]
```diff
@@ -286,7 +286,7 @@
 #	STATE IS TO BE UPDATED ALONG A CONIC TRAJECTORY, CALCULATES THE CORRESPONDING TIME-OF-FLIGHT AND, IN ADDITION,
 #	PROVIDES THE OPTION OF COMUTING THE NEW UPDATED STATE VECTOR.  THE RESULTING TRAJECTORY MAY BE A SECTION OF A
 #	CIRCLE, ELLIPSE, PARABOLA, OR HYPERBOLA WITH RESPECT TO THE EARTH OR THE MOON.  THE USE OF THE SUBROUTINE CAN BE
-# 	EXTENDED USING OTHER PRIMARY BODIES BY SIMPLE ADDITIONS TO THE MUTABLE WTIHOUT INTRODUCING ANY CODING CHANGES,
+# 	EXTENDED USING OTHER PRIMARY BODIES BY SIMPLE ADDITIONS TO THE MUTABLE WITHOUT INTRODUCING ANY CODING CHANGES,
```
Comment (marcosalles) [inline CONIC_SUBROUTINES.s]: ok, what's wrong with `WITH`?

File: CSM_GEOMETRY.s [hunk 4]
```diff
@@ -33,7 +33,7 @@
 		BANK
 		
 # THIS ROUTINE TAKES THE SHAFT AND TRUNNION ANGLES AS READ BY THE CM OPTICAL SYSTEM AND CONVERTS THEM INTO A UNIT
-# VECTOR REFERENCED TO THE NAVIGATION BASE COORDINATE SYSTEM AND COINCIDENT WTIH THE SEXTANT LINE OF SIGHT.
```
Comment (marcosalles) [inline CSM_GEOMETRY.s]: seriously, do you not like the word `WITH`?

File: DOWN-TELEMETRY_PROGRAM.s [hunk 5]
```diff
@@ -122,7 +122,7 @@
 #		SAME AS ECADR, BUT USED WHEN THE WORD ADDRESSED IS THE LEFT
 #		HALF OF A DOUBLE-PRECISION WORD FOR DOWN TELEMETRY.
 #	B.	2DNADR - 6DNADR		N-WORD DOWNLIST ADDRESS, N = 2 - 6.
-#		SAME AS 1DNADR, BUT WTIH THE 4 UNUSED BITS OF THE ECADR FORMAT
```
Comment (marcosalles) [inline DOWN-TELEMETRY_PROGRAM.s]: that's starting to look like prejudice

File: DOWN_TELEMETRY_PROGRAM.s [hunk 6]
```diff
@@ -115,7 +115,7 @@
 #		SAME AS ECADR, BUT USED WHEN THE WORD ADDRESSED IS THE LEFT
 #		HALF OF A DOUBLE-PRECISION WORD FOR DOWN TELEMETRY.
 #	B.	2DNADR - 6DNADR		N-WORD DOWNLIST ADDRESS, N = 2 - 6.
-#		SAME AS 1DNADR, BUT WTIH THE 4 UNUSED BITS OF THE ECADR FORMAT
```
Comment (marcosalles) [inline DOWN_TELEMETRY_PROGRAM.s]: ok, is this a joke?

File: FRESH_START_AND_RESTART.s [hunk 7]
```diff
@@ -99,7 +99,7 @@
 #
 #	1.	INTIALIZE OUTBIT CHANNELS 11,12,13, AND 14
 #	2.	REPLACE ALL TASKS ON WAITLIST WITH ENDTASK
```
Comment (marcosalles) [inline FRESH_START_AND_RESTART.s]: why is this one right, but so many mistakes?

File: FRESH_START_AND_RESTART.s [hunk 8]
```diff
@@ -130,7 +130,7 @@
 #	B.	ALARMS
 #
 #		1107	PHASE TABLE ERROR
-#		1110	RESTART WTIH NO ACTIVE GROUPS
```
Comment (marcosalles) [inline FRESH_START_AND_RESTART.s]: I give up

File: P32-P35_P72-P75.s [hunk 9]
```diff
@@ -34,7 +34,7 @@
 #
 # PURPOSE
 
-#	(1)	TO CALCULATE PARAMETERS ASSOCIATED WTIH THE TIME FOLLOWING
```
Comment (marcosalles) [inline P32-P35_P72-P75.s]: Ok I don't give up. How can someone misspell the same word so many times?

File: P37_P70.s [hunk 10]
```diff
@@ -44,7 +44,7 @@
 # DESCRIPTION
 #	A RETURN TO EARTH TRAJECTORY IS COMPUTED PROVIDED THE CSM IS OUTSIDE THE LUNAR SPHERE OF INFLUENCE AT THE
 #	TIME OF IGNITION.  INITIALLY A CONIC TRAJECTORY IS DETERMINED AND RESULTING IGNITION AND REENTRY PARAMETERS ARE
-# 	DISPLAYED TO THE ASTRONAUT.  THEN IF THE ASTRONAUT SO DESIRES, A PRECISION TRAJECTORY IS DETERMINED WTIH THE
```
Comment (marcosalles) [inline P37_P70.s]: What
The
Idiotic
Hell

File: PINBALL_GAME_BUTTONS_AND_LIGHTS.s [hunk 11]
```diff
@@ -2817,7 +2817,7 @@ SETVAC		CAF	TCFINDVC
 		
 # VBRQWAIT ENTERS REQUEST TO WAITLIST FOR ANY ADDRESS WITH ANY DELAY.
 # IT DOES ENDOFJOB AFTER ENTERING REQUEST.  DISPLAY SYST IS RELEASED.
-# IT ASSUMES NOUN 26 HAS BEEN PRELOADED WTIH
```
Comment (marcosalles) [inline PINBALL_GAME_BUTTONS_AND_LIGHTS.s]: What @radarhere said.

File: PINBALL_GAME_BUTTONS_AND_LIGHTS.s [hunk 12]
```diff
@@ -2921,11 +2921,11 @@ ENDRELDS	EQUALS
 # PLACE 0VVVVVVVNNNNNNN INTO A.
 # V'S ARE THE 7-BIT VERB CODE.  N'S ARE THE 7-BIT NOUN CODE.
 #
-# IF NVSUB IS CALLED WTIH THE FOLLOWING NEGATIVE NUMBERS (RATHER THAN THE
```
Comment (marcosalles) [inline PINBALL_GAME_BUTTONS_AND_LIGHTS.s]: Ok, that's awkward

File: PINBALL_GAME_BUTTONS_AND_LIGHTS.s [hunk 13]
```diff
@@ -2921,11 +2921,11 @@ ENDRELDS	EQUALS
...
+# IF NVSUB IS CALLED WITH THE FOLLOWING NEGATIVE NUMBERS (RATHER THAN THE
 # VERB-NOUN CODE) IN A, THEN THE DISPLAY IS BLANKED AS FOLLOWS ---
 # -4 FULL BLANK, -3 LEAVE MODE, -2 LEAVE MODE AND VERB, -1 BLANK R'S ONLY.
 #
-# NVSUB CAN BE USED WTIH MACHINE CADR TO BE SPECIFIED BY PLACING THE CADR INTO
```
Comment (marcosalles) [inline PINBALL_GAME_BUTTONS_AND_LIGHTS.s]: I'm seriouslly thinking about giving up for good.

File: PINBALL_GAME_BUTTONS_AND_LIGHTS.s [hunk 14]
```diff
@@ -2960,7 +2960,7 @@ ENDRELDS	EQUALS
 # (SIMILARLY FOR PLEASE MARK).  FIRST PLACE THE CODED NUMBER FOR WHAT
 # ACTION IS DESIRED OF OPERATOR INTO THEREGISTERS REFERRED TO BY THE
 # `CHECKLIST' NOUN.  GO TO NVSUB WITH A DISPLAY VERB AND THE `CHECKLIST'
-# NOUN.  GO TO NVSUB AGAIN WTIH THE `PLEASE PERFORM' VERB AND ZEROS IN THE
+# NOUN.  GO TO NVSUB AGAIN WITH THE `PLEASE PERFORM' VERB AND ZEROS IN THE
```
Comment (marcosalles) [inline PINBALL_GAME_BUTTONS_AND_LIGHTS.s]: Dude, you're wrong. It's `WTIH`

File: PINBALL_NOUN_TABLES.s [hunk 15]
```diff
@@ -120,7 +120,7 @@
 # (WHICH ARE IN A SEPARATE BANK FROM THE REST OF PINBALL).  THESE READING
 # ROUTINES ARE IN THE SAME BANK AS THE TABLES.  THEY ARE CALLED BY DXCH Z.
 
-# LODNNTAB LOADS NNADTEM WTIH THE NNADTAB ENTRY, NNTYPTEM WITH THE
+# LODNNTAB LOADS NNADTEM WITH THE NNADTAB ENTRY, NNTYPTEM WITH THE
```
Comment (marcosalles) [inline PINBALL_NOUN_TABLES.s]: The evidence is there, don't try to deny it.

File: PINBALL_NOUN_TABLES.s [hunk 16]
```diff
@@ -165,7 +165,7 @@ LODNLV		DXCH	IDAD2TEM		# PUT RETURN INFO INTO A, L.
 		
 MIXCON		=	OCT50			# FIRST MIXED NOUN =40.  (DEC 40)
 
-# GTSFOUT LOADS SFTEMP1, SFTEMP2 WTIH THE DP SFOUTAB ENTRIES.
+# GTSFOUT LOADS SFTEMP1, SFTEMP2 WITH THE DP SFOUTAB ENTRIES.
```
Comment (marcosalles) [inline PINBALL_NOUN_TABLES.s]: See? No way someone could write something wrong so many times.

File: PINBALL_NOUN_TABLES.s [hunk 17]
```diff
@@ -214,7 +214,7 @@ NNADTAB		OCT	00000			# 00 	NOT IN USE
 		OCT	00000			# 23	SPARE
 		ECADR	DSPTEM2 +1		# 24	DELTA TIME FOR AGC CLOCK (HRS,MIN,SEC)
 		ECADR	DSPTEM1			# 25	CHECKLIST
-						#	(USED WTIH PLEASE PERFORM ONLY)
+						#	(USED WITH PLEASE PERFORM ONLY)
```
Comment (marcosalles) [inline PINBALL_NOUN_TABLES.s]: `WTIH`

File: T4RUPT_PROGRAM.s [hunk 18]
```diff
@@ -577,7 +577,7 @@ NXTFL33		CCS	RUPTREG2		# PROCESS POSSIBLE ADDITIONAL CHANGES.
 #
 # ERASABLE INITIALIZATION:
 #		1) FRESH START OR RESTART WITH NO GROUPS ACTIVE:  C(CDUZ) = 0, IMODES30 BIT 6 = 0, IMODES33 BIT 1 = 0.
-#		2) RESTART WTIH GROUPS ACTIVE:	SAME AS FRESH START EXCEPT C(CDUZ) NOT CHANGED SO GIMBAL MONITOR
+#		2) RESTART WITH GROUPS ACTIVE:	SAME AS FRESH START EXCEPT C(CDUZ) NOT CHANGED SO GIMBAL MONITOR
```
Comment (marcosalles) [inline T4RUPT_PROGRAM.s]: WTIH. You clearly don't know how to write this word, neither did I until now.

File: TVCEXECUTIVE.s [hunk 19]
```diff
@@ -100,7 +100,7 @@ ROLLPREP	CAE	CDUX		# UPDATE ROLL LADDERS (NO NEED TO RESTART-
 		XCH	OGANOW		# 	PROTECT, SINCE ROLL DAPS RE-START)
 		XCH	OGAPAST
 		
-		CAE	OGAD		# PREPARE ROLL FDAI NEEDLE WTIH FLY-TO
+		CAE	OGAD		# PREPARE ROLL FDAI NEEDLE WITH FLY-TO
```
Comment (marcosalles) [inline TVCEXECUTIVE.s]: I guess it's useless to argue wtih a pull-request, but I do hope you read and fix this.

**Comment by chrislgarry** [comment]
Need to investigate which typos are in the original source. If this is a typo of the person who digitized, I will merge it in.

**Comment by karuna** [comment]
I'm pretty sure there's a reason to use `WTIH`

File: BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s (hunk 1 above)
Comment (radarhere) [inline BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s]: Thanks. Nice catch. Fixed it in a subsequent commit.

**Comment by radarhere** [comment]
@chrislgarry thanks. I did wonder whether the purpose of this repo was simply to be a historical archive.

**Comment by darcy** [comment]
Hmm - I can't get the repo to build with these changes. Can you?

File: DOWN_TELEMETRY_PROGRAM.s (hunk 6 above)
Comment (Kangaroonicorn) [inline DOWN_TELEMETRY_PROGRAM.s]: This explains the bug in version 13

File: KALCMANU_STEERING.s [hunk 20]
```diff
@@ -89,7 +89,7 @@ INCRDCDU	TS	KSPNDX
 		TCF	INCRDCDU	# LOOP FOR THREE AXES
 		
 		RELINT		
-# COMPARE PRESENT TIME WTIH TIME TO TERMINATE MANEUVER
```
Comment (klarkc) [inline KALCMANU_STEERING.s]: TROLL!

File: P32-P33_P72-P73.s [hunk 21]
```diff
@@ -42,7 +42,7 @@
 # MOD BY WHITE, P.	DATE 1 JUNE 67
 #
 # PURPOSE
-#	(1)	TO CALCULATE PARAMETERS ASSOCIATED WTIH THE FOLLOWING
```
Comment (klarkc) [inline P32-P33_P72-P73.s]: STAWP!

File: TVCEXECUTIVE.s [hunk 22]
```diff
@@ -100,7 +100,7 @@ ROLLPREP	CAE	CDUX		# UPDATE ROLL LADDERS (NO NEED TO RESTART-
 		XCH	OGANOW		# 	PROTECT, SINCE ROLL DAPS RE-START)
 		XCH	OGAPAST
 		
-		CAE	OGAD		# PREPARE ROLL FDAI NEEDLE WTIH FLY-TO
```
Comment (klarkc) [inline TVCEXECUTIVE.s]: CTRL + H, replace all WTIH with WITH quickly, you are welcome.

**Comment by keikun17** [comment]
LTGM

File: TVCEXECUTIVE.s (hunk 22 above)
Comment (JetStream96) [inline TVCEXECUTIVE.s]: But, but are you sure that won't break the code. These typos must be dealt wtih more carefully.

**Comment by mmajko** [comment]
I like what have you done wtih this code.

**Comment by padi** [comment]
Wtih great power comes great responsibility.

File: KEYRUPT_UPRUPT.s [hunk 23]
```diff
@@ -127,7 +127,7 @@ ELRCODE		OCT	22
 # THE RECEPTION OF A BAD CODE (I.E., CCC FAILURE) LOCKS OUT FURTHER UPLINK ACTIVITY BY SETTING BIT4 OF FLAGWRD7 = 1.
 # THIS INDICATION WILL BE TRANSFERRED TO THE GROUND BY THE DOWNLINK WHICH DOWNLINKS ALL FLAGWORDS.
 # WHEN UPLINK ACTIVITY IS LOCKED OUT, IT CAN BE ALLOWED WHEN THE GROUND UPLINKS AND `ERROR RESET' CODE.
-# (IT IS RECOMMENDED THAT THE `ERROR LIGHT RESET' CODE IS PRECEEDED BY 16 BITS THE FIRST OF WHICH IS 1 FOLLOWED
+# (IT IS RECOMMENDED THAT THE `ERROR LIGHT RESET' CODE IS PRECEDED BY 16 BITS THE FIRST OF WHICH IS 1 FOLLOWED
```
Comment (oznogon) [inline KEYRUPT_UPRUPT.s]: This typo is present in the scan of [page 1451](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1451.jpg).

File: P20-P25.s [hunk 24]
```diff
@@ -420,7 +420,7 @@ DE-GR-50	TC	2PHSCHNG
 #
 # ABORT MODES:  P23 ABORT IF MARKING SYSTEM OR EXTENDED VERB ACTIVE
 #
-# INPUT:  NONE REQURIES, NORMALLY CALLED BY P23
+# INPUT:  NONE REQUIRES, NORMALLY CALLED BY P23
```
Comment (oznogon) [inline P20-P25.s]: Based on the [source scan of page 570](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0570.jpg), this line should read:

```
# INPUT:  NONE REQUIRED. NORMALLY CALLED BY P23
```

File: Q_R-AXIS_RCS_AUTOPILOT.s [hunk 25]
```diff
@@ -476,7 +476,7 @@ QERRCALC	CAE	CDUY		# Q-ERROR CALCULATION
 		TS	DAPTEMP2	# SAVE FOR RERRCALC
 		EXTEND
 		MP	M22		# (CDUZ-CDUZD)*M22 SCALED AT PI RADIANS
-		AD	DELQEROR	# KALCMANU INERFACE ERROR
+		AD	DELQEROR	# KALCMANU INTERFACE ERROR
```
Comment (oznogon) [inline Q_R-AXIS_RCS_AUTOPILOT.s]: This is `INERFACE` in the scan of [page 1451](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1451.jpg).

File: Q_R-AXIS_RCS_AUTOPILOT.s [hunk 26]
```diff
@@ -487,7 +487,7 @@ RERRCALC	CAE	DAPTEMP1	# R-ERROR CALCULATION:
 		CAE	DAPTEMP2	# SECOND TERM CALCULATION:
 		EXTEND			# CDU ANGLE -ANGLE DESIRED (Z-AXIS)
 		MP	M32		# (CDUZ-CDUZD)*M32 SCALED AT PI RADIANS
-		AD	DELREROR	# KALCMANU INERFACE ERROR
+		AD	DELREROR	# KALCMANU INTERFACE ERROR
```
Comment (oznogon) [inline Q_R-AXIS_RCS_AUTOPILOT.s]: This is `INERFACE` in the scan of [page 1451](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1451.jpg).

File: RCS-CSM_DIGITAL_AUTOPILOT.s [hunk 27]
```diff
@@ -295,7 +295,7 @@ TENTHSEK	TS	ATTSEC
 #	    ALSO IGNORE AUTOMATIC STEERING
 #	    SET = + BY
 #		1) INITIALIZATION PHASE OF AUTOPILOT
-#		2) OCCURANCE OF RHC COMMANDS
+#		2) OCCURRENCE OF RHC COMMANDS
```
Comment (oznogon) [inline RCS-CSM_DIGITAL_AUTOPILOT.s]: This is `OCCURANCE` in the scan of [page 1007](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1007.jpg).

File: RT8_OP_CODES.s [hunk 28]
```diff
@@ -136,7 +136,7 @@ TPMODE		CAF	ONE		# MODE IS TP.
 # Page 1510
 		TC	Q
 
-		INDEX	A		# OVERFLOW UNCORRECT AND IN MSU.
+		INDEX	A		# OVERFLOW INCORRECT AND IN MSU.
```
Comment (oznogon) [inline RT8_OP_CODES.s]: This is `UNCORRECT` in the scan of [page 1510](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1510.jpg).

File: RTB_OP_CODES.s [hunk 29]
```diff
@@ -113,7 +113,7 @@ TPMODE		CAF	ONE		# MODE IS TP.
 		TS	MPAC		# AND SKIP ON OVERFLOW.
 		TC	Q
 
-		INDEX	A		# OVERFLOW UNCORRECT AND IN MSU.
+		INDEX	A		# OVERFLOW INCORRECT AND IN MSU.
```
Comment (oznogon) [inline RTB_OP_CODES.s]: This is `UNCORRECT` on the scan of [page 1398](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1398.jpg).

File: UPDATE_PROGRAM.s [hunk 30]
```diff
@@ -122,7 +122,7 @@
 #		V33E		VERB 33 TO SIGNAL THAT THE STATE VECTOR IS READY TO BE STORED.
 #
 #	2.  REFSMMAT (ALL DATA ENTRIES IN OCTAL)
-#		ENTRIES		DATA DEFINITITIONS				SCALE FACTORS:
+#		ENTRIES		DATA DEFINITIONS				SCALE FACTORS:
```
Comment (oznogon) [inline UPDATE_PROGRAM.s]: Per the scan of [page 1498](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1498.jpg), this line should read:

```
#		ENTRIES:		DATA DEFINITIONS:			SCALE FACTORS:
```

**Comment by oldmud0** [comment]
Don't worry, there are more to come

**Comment by oznogon** [comment]
@radarhere I've noted where some of the typos exist or are otherwise different in the original source scans.

**Comment by oldmud0** [comment]
I've applied this patch to my clone of the repo and am actually looking through the source myself for more typos. About 75% done right now. I'll probably make a PR tomorrow about it.

**Comment by oznogon** [comment]
@oldmud0 Please check your fixes against the scanned pages for [Comanche 055](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/) and [Luminary 099](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099) to confirm that the typos exist only in the transcription.

**Comment by oldmud0** [comment]
@oznogon OK, will do once I revise the final diff.

**Comment by chrislgarry** [comment]
@radarhere some of these are typos, and some are not. please resolve the merge conflicts, and submit a PR with verified typos. Thanks. @oldmud0 @oznogon thanks for the help!

**Comment by radarhere** [comment]
Thanks very much to @oznogon @oldmud0 and @chrislgarry. Thanks for the positive comments from others as well!

This has since been incorporated into #91, so closing.

**Pull request closed without merging**